src/components/accordion/accordion.component.js

```diff
@@ -163,6 +162,7 @@ const Accordion = React.forwardRef(({
 });
 
 Accordion.propTypes = {
+  ...propTypes.space,
```

Comment:
Could you add this comment above this line please: `/** Styled system spacing props */` ? 
Just so it's immediately obvious what this is